components/support/images/src/main/res/values/dimens.xml

```diff
@@ -2,6 +2,7 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <dimen name="mozac_support_images_size_default">102dp</dimen>
+    <dimen name="mozac_support_images_size_max" tools:ignore="PxUsage">2153px</dimen>
```

Comment:
Is there any specific reason why the unit are not the same as `mozac_support_images_size_default`? :)

Reply:
I copied it from [here](https://github.com/mozilla-mobile/android-components/blob/master/components/browser/thumbnails/src/main/res/values/dimens.xml). Seems to be the hard coded maximum px size for thumbnails.